package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "bluebird": "^3.3.4",
     "htmlencode": "^0.0.4",
-    "lodash.merge": "^4.6.1",
+    "lodash": "^4.17.11",
```

Comment:
Hey @daniel-cottone - thanks so much for submitting this!

This is a pretty big jump, could this potentially break anything?

Reply:
@jonnyom I ran the tests and they all passed; I don't think the core functionality of `merge` has changed much as well.

Reply:
OK cool, nice one!